definitions/ext-pixie_dns/definition.yml

```diff
@@ -0,0 +1,34 @@
+domain: EXT
+type: PIXIE_DNS
+synthesis:
+  rules:
+  - identifier: dns.dns_server
```

Comment:
As we also use this attribute for the name, it seems to be human-readable.
Is this unique [enough] ?

Could I create two dns servers with the same `dns.dns_server` attribute within a customer account ?

Reply:
Yeah thinking about this more, I think this has to be more specific for the cluster. Even if that means some dns servers are not correctly attributed.